metron-deployment/packaging/ambari/metron-mpack/pom.xml

```diff
@@ -60,6 +64,19 @@
                             <failIfNoMatch>false</failIfNoMatch>
                         </configuration>
                     </execution>
+                    <execution>
+                        <id>metron-short-version</id>
+                        <goals>
+                            <goal>regex-property</goal>
+                        </goals>
+                        <configuration>
+                            <name>metron.short.version</name>
+                            <value>${project.parent.version}</value>
+                            <regex>\.</regex>
```

Comment:
@mmiklavc Did you ever have a chance to figure this out?  Given that we haven't agreed on version yet anyway, it may not matter too much, but I would be delighted if "-SNAPSHOT" worked.